src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/ErrorHandler.kt

```diff
@@ -64,7 +64,7 @@ class ErrorHandler(templateEngine: TemplateEngine)
     {
         logger.warn("For request ${req.uri()}, a ${error::class.simpleName} occurred with the following problems: ${error.problems}")
 
-        if (req.pathInfo().startsWith(Router.apiUrlBase))
+        if (req.pathInfo().startsWith(Router.apiUrlBase) || req.headers("Accept").contains("application/json"))
```

Comment:
Not sure if this the best way of dealing with an issue I had where if an error was thrown by an 'api-like web endpoint' (ie the endpoints that are only supposed to be ajax calls from the portal rather than part of the public api), we were getting an html error response because of this line making a decision based on the url path only.

Reply:
Ah of course. This seems like the right way to go.